.github/workflows/build.yml

```diff
@@ -11,63 +11,78 @@ jobs:
     runs-on: windows-latest
     steps:
     - uses: actions/checkout@v1
-    - name: Build (Lightweight)
-      run: |
-        cd selene
-        cargo build --locked --release --verbose --no-default-features
-        New-Item release -ItemType "directory"
-        Move-Item ..\target\release\selene.exe .\release\selene-light.exe
     - name: Build (All features)
       run: |
         cd selene
         cargo build --locked --release --all-features
+        New-Item release -ItemType "directory"

```

Comment:
If we're going to be just shipping out the executables directly, I don't think we need to be making directories anymore--you can just send straight from target/release.

Reply:
Done! I would've preferred to keep the build steps and the upload steps grouped, but this seems faster.